core-it-suite/src/test/resources/mng-6511-optional-project-selection/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.apache.its.mng6511</groupId>
+  <artifactId>parent</artifactId>
+  <version>1.0</version>
+
+  <packaging>pom</packaging>
+
+  <name>Maven Integration Test :: MNG-6511</name>
+  <description>
+    Tests for selecting and deselecting optional projects using the question mark classifier.
+  </description>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+  </properties>
+
+  <modules>
+    <module>existing-module</module>
+  </modules>
+
+  <build>
+    <pluginManagement>
+      <!-- maven-resources-plugin, maven-compiler-plugin -->
+      <plugins>
```

Comment:
Did you check whether these are downloaded via bootstrap IT? If not, use the same versions as bundled with Maven 4.

Reply:
Yep, [they are](https://github.com/apache/maven-integration-testing/blob/master/core-it-suite/src/test/resources/bootstrap/pom.xml#L61-L80) :)